test/cluster/test_repair.py

```
@@ -450,6 +450,98 @@ async def test_repair_rejects_equal_start_and_end_token(manager):
                                            host=servers[0].ip_addr, params=params)
 
 
+async def test_repair_batched_stream_rack_compression_and_dict(manager: ManagerClient) -> None:
```

Ruff (0.16.3)

[error] Undefined name `ManagerClient`

(F821)